components/dialog/skin.css

```diff
@@ -1,24 +0,0 @@
-/*!
-Copyright 2023 Adobe. All rights reserved.
-This file is licensed to you under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License. You may obtain a copy
-of the License at http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software distributed under
-the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
-OF ANY KIND, either express or implied. See the License for the specific language
-governing permissions and limitations under the License.
-*/
-
-.spectrum-Dialog-heading {
-  color: var(--spectrum-dialog-confirm-title-text-color);
-}
-
-.spectrum-Dialog-content {
-  color: var(--spectrum-dialog-confirm-description-text-color);
-}
-
-.spectrum-Dialog-footer {
-  color: var(--spectrum-dialog-confirm-description-text-color);
-}
-
```

Comment:
So it looks like these were just deleted rather than moved over into index.css which is making the text color incorrect. Could you add these back in to index.css?

Reply:
done!
I  don't know how i forgot this!